### PR TITLE
added toSentence

### PR DESCRIPTION
**Pull request by alexvandesande**
Changed camel cased events

### CODE REVIEW

File: app/client/lib/helpers/helperFunctions.js [hunk 1]
```diff
@@ -355,3 +355,17 @@ Helpers.addInputValue = function (inputs, currentInput, formField){
...
 };
+
+/**
+Takes a camelcase and shows it with spaces
+
+@method toSentence
+@param {string} camelCase    A name in CamelCase or snake_case format
+@return {string} sentence    The same name with spaces
+**/
+Helpers.toSentence = function (name, noHTML) {
```
Comment (frozeman) [inline app/client/lib/helpers/helperFunctions.js]: i would rather make the HTML true and see the noHTML as default

Comment (alexvandesande) [inline app/client/lib/helpers/helperFunctions.js]: The logic was that there are many more instances where HTML is used, as noHTML is used only inside <option> tags, and therefore the parameter can be omitted in the most common use.

Comment (frozeman) [inline app/client/lib/helpers/helperFunctions.js]: Hm i personally would do it the other way around, as noHTML is "native" HTML is an feature.

E.g. i order to plan for the future, i would make a `Helper.toSentence` ja function, which is can use anywhere and then atop, make it also a template helper. In this case the general helper will never use `toHTML`. Its only interesting for the template helper. Thats why i would make the `noHTML` the default...

File: app/client/templates/elements/eventsTable.js [hunk 2]
```diff
@@ -59,15 +59,16 @@ Template['elements_event_table'].helpers({
...
 
                 // search to address
                 if(pattern.test(item.address))
-                    return item;
+                    return _.extend(item, {name: 'unnamed'});
 
                 // search to return values
                 if(_.find(item.args, function(value, name){ return pattern.test(value) || pattern.test(name); }))
-                    return item;
+                    return _.extend(item, {name: 'args'});
```
Comment (frozeman) [inline app/client/templates/elements/eventsTable.js]: What not passing the name here too? like below with `item.event`

Comment (alexvandesande) [inline app/client/templates/elements/eventsTable.js]: yeah, both should bee the i18n "Contract Execution"

**Pull request merged**